src/pages/Wishlist/CourseWishlistCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import "./CourseWishlistCard.css";
+
+const CourseWishlistCard = ({ product }) => {
+  return (
+    <article className="course-wishlist-card">
+      <div className="img-wrapper">
+        <img src={product.img} alt={product.title} />
+      </div>
+      <div className="content h4">
+        <Link to="/course" className="link">
+          {product.title}
+        </Link>
+      </div>
+      <div className="fw-b price">₹{product.originalPrice}</div>
```

Comment:
You could destructure the product here.
Line 23 below has used class instead of className. Please check.

The UI looks good, very neat!

Reply:
I will look into it. Also thanks for the compliment